.github/workflows/ci-actions-incremental.yml

```diff
@@ -908,8 +908,8 @@ jobs:
         if: startsWith(matrix.os-name, 'windows')
         with:
           version: 'mandrel-latest'
```

Comment:
The `version` should stay there?

Reply:
Yes `setup-graalvm` requires a version for the mandrel distribution, I have a fix for it but I would like to incorporate in a fix for https://github.com/graalvm/setup-graalvm/issues/64 so that it's complete.